VS/VehicleSignals.rdf

```diff
@@ -2336,6 +2337,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;IsBackUpLightOn">
 		<rdfs:subClassOf rdf:resource="&auto-vs;ActuableSignal"/>
+		<rdfs:label xml:lang="en">Is backup light on</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.